compiler/rustc_target/src/spec/targets/thumbv7em_m4_none_eabi.rs

```diff
@@ -0,0 +1,10 @@
+use crate::spec::{Target, TargetMetadata};
+
+pub(crate) fn target() -> Target {
+    let mut target = super::thumbv7em_none_eabi::target();
```

Comment:
@jonathanpallant could you help me figure out what specific options we might want to set here to optimize for the M4 CPU?

Reply:
For the Cortex-M4, target-cpu=cortex-m4 is the only one I know of.